dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/context/annotation/DubboClassPathBeanDefinitionScanner.java

```diff
@@ -53,7 +53,7 @@ public DubboClassPathBeanDefinitionScanner(BeanDefinitionRegistry registry, bool
     public DubboClassPathBeanDefinitionScanner(BeanDefinitionRegistry registry, Environment environment,
                                                ResourceLoader resourceLoader) {
 
-        this(registry, false, environment, resourceLoader);
+        this(registry, true, environment, resourceLoader);
```

Comment:
why change it from false to true

Reply:
Supports reference bean auto-discovery using component annotations.

Reply:
When we use EnableDubbo annotation, we can remove @ComponentScan annotation now, you can see com.alibaba.dubbo.examples.annotation.AnnotationConsumer commit history.

Reply:
Supports @Service to automatically discover Component annotations

Reply:
@zonghaishang DO NOT implement , it will scan all `@Component` beans

Reply:
com.alibaba.dubbo.config.spring.beans.factory.annotation.ServiceAnnotationBeanPostProcessor#registerServiceBeans:

```
        scanner.addExcludeFilter(new AnnotationTypeFilter(Component.class, false));
        scanner.addExcludeFilter(new AnnotationTypeFilter(Controller.class, false));
        scanner.addExcludeFilter(new AnnotationTypeFilter(Repository.class, false));
```

I have filtered that, @EnableDubbo should support client scanning.